apps/web/src/components/features/properties/PropertyImageGallery.tsx

```
@@ -0,0 +1,256 @@
+        </div>
+
+        {/* Thumbnail Strip */}
+        {images.length > 1 && (
+          <div className="flex gap-2 mt-4 overflow-x-auto pb-2">
+            {images.map((image, index) => {
+              // Create a unique key using image URL hash or fallback to index
+              // const imageKey = `thumbnail-${image.replace(/[^a-zA-Z0-9]/g, "")}-${index}`
+
+              return (
+                <button
+                  key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)